config/rbac/role.yaml

```diff
@@ -483,9 +451,6 @@ rules:
   - secrets
   verbs:
   - '*'
-  - create
-  - get
-  - watch
```

Comment:
Not related to this PR per se, but since you touched it maybe worth narrowing it from `*` to relevant verbs? For both `secrets` and `oauthclients`

Reply:
yep, replaced with verbs first then we can later narrow down more